lepton/aws.go

```diff
@@ -672,8 +672,18 @@ func (p *AWS) DeleteInstance(ctx *Context, instancename string) error {
 	return nil
 }
 
+// PrintInstanceLogs writes instance logs to console
+func (p *AWS) PrintInstanceLogs(ctx *Context, instancename string, watch bool) error {
+	l, err := p.GetInstanceLogs(ctx, instancename)
```

Comment:
@fabioDMFerreira I just wonder why do we need to it this way. I see across all providers we have the same implementation for the method. Should we just handle it at `cmd/instance.go`?
Ideally, we should avoid printing logs from providers or common logs. Consumers of the method can decide if to print logs or not.

Reply:
I did the same question while coding @sanketsudake.

We have a `watch` flag (still not supported in any provider) in the command that should listen for new changes in the log and print them in real-time.  This implementation may be different from provider to provider.

Ideally, we should return a read stream, but for simplicity reasons, we're returning a string.

Reply:
Created a new ticket for this #702.

Reply:
I think the main reason for keeping it in provider specific methods is cause of the tail/watch functionality, gcp has this functionality && vsphere could have it, I think azure has support for it too, so prob. worth keeping as-is even if it seems a bit redundant for now